README.md

```diff
@@ -235,6 +253,22 @@ Type comprising every "plain" Object value. Specifically, values created via:
   - the `new` operator in conjunction with `Object` or a custom
     constructor function.
 
+#### `PositiveInteger`
+
+```haskell
+$.PositiveInteger :: Type
+```
+
+Type comprising every [`Integer`](#integer) value greater than zero.
+
+#### `NegativeInteger`
+
+```haskell
+$.NegativeInteger :: Type
+```
+
+Type comprising every [`Integer`](#integer) value less than zero.
```

Comment:
Yep, I agree. That was just a mistake on my part. The list was meant to be in alphabetical order.